frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/components/Tooltip/index.tsx

```diff
@@ -53,9 +53,7 @@ const ERROR_TO_CONTENT: { [key: string]: Content } = {
   },
   [FAILED_RETURN_TYPE.UPLOAD + UPLOAD_STATUS.FAILED]: {
     color: RED.C,
-    content: (
-      <span>There was a problem uploading your file. Please try again.</span>
-    ),
+    content: <span>There was a problem uploading your file.</span>,
```

Comment:
Do we want to say something to communicate:
> CZI will take a look at it and reach out with any updates, here is our SLA

cc: @maniarathi

Reply:
Let's leave it as is for now as we test out our SLA for a few on-call cycles before committing publicly.